tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -758,6 +761,8 @@ public void ttlCreateFileTest() throws Exception {
     Assert.assertEquals(ttl, folderInfo.getTtl());
   }
 
+  @LocalAlluxioClusterResource.Config(
```

Comment:
I think the previous convention is to have `@Test` first,
then `@LocalAlluxioClusterResource.Config`